packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -95,7 +95,7 @@
 +## Relies on the existence of a wins server of version v0.1.0+ on every Windows host to allow 
 +## windows_exporter to run as a host process that can publish host metrics to a port on the Pod
 +windowsExporter:
-+  enabled: true
++  enabled: false
```

Comment:
We want this to be false now? What's the reasoning?

Reply:
by default, it should be false since it is expected it will be deployed in a non-Windows cluster. I'll be creating a UI ticket for Windows to enable this by default automatically only if the cluster is detected to be a Windows cluster